zerver/models.py

```diff
@@ -397,6 +397,12 @@ def upload_quota_bytes(self) -> Optional[int]:
         # it as gibibytes (GiB) to be a bit more generous in case of confusion.
         return self.upload_quota_gb << 30
 
+    def currently_used_upload_space_bytes(self) -> int:
```

Comment:
We should probably make this function have a cache that lasts an hour, so that it doesn't become a significant performance bottleneck.